upgrade_analysis/compare.py

```diff
@@ -121,8 +128,10 @@ def fieldprint(old, new, field, text, reprs):
             text += f" [{old['table']}]"
         if field == "relation":
             text += " [nothing to do]"
-    reprs[module_map(old["module"])].append(f"{fullrepr}: {text}")
+    if field != "module":
+        reprs[module_map(new["module"])].append(f"{fullrepr}: {text}")
     if field == "module":
+        reprs[module_map(old["module"])].append(f"{fullrepr}: {text}")
```

Comment:
FYI, I did this change now for one reason:

If you look into https://github.com/OCA/OpenUpgrade/pull/4918, you see that the "now required" is shown in `sale_project`, but I think it's better if it's shown in `product`.